### PR TITLE
h2 ccgt

### PR DESCRIPTION
**Pull request by oruhnau**
I overrided fuel cell parameters with those of CCGT, which is more favorable.

Some H2 CCGT are build at the outer edges of the network.

Could be done more nicely when we decide to keep this I think.

### CODE REVIEW

**Comment by oruhnau** [comment]
These are my results:

grafik

**Comment by oruhnau** [comment]
Interestingly, H2 storage is non-zero:
grafik

And there are quite some pipelines being built:
grafik

**Comment by oruhnau** [comment]
@lisazeyen 

I found a (rather old) source on the efficiency of H2 turbines, claiming that: "Durch den Einsatz von Wasserstoff als Brennstoff ist keine signifikante Verschlechterung des Wirkungsgrads zu erwarten."
[Stolzenburg et al. - 2014 - Integration von Wind-Wasserstoff-Systemen in das E.pdf](https://github.com/lisazeyen/hydrogen_policy/files/13347699/Stolzenburg.et.al.-.2014.-.Integration.von.Wind-Wasserstoff-Systemen.in.das.E.pdf)

Not so easy to find something on costs. This is the only hint so far: labels on H2-readiness, based on 5-20% retrofitting costs. But it remains unclear whether any turbine has been labeled yet, and this is not exactly the same as new-built turbines...
grafik

I suggest moving on (i.e. merging) as is and keeping an eye open for better assumptions

**Comment by oruhnau** [comment]
I finally found cost assumptions for hydrogen turbines:
grafik
Quelle: https://energiesysteme-zukunft.de/fileadmin/user_upload/Publikationen/PDFs/ESYS_Analyse_IntEv.pdf, Seite 194

In 2030, H2-OCGTs are 10% more expensive than OCGTs running on natural gas. With CCGTs, the cost difference should be smaller - I assumed 5%.

I also ran the reference scenario with H2-OCGTs instead of H2-CCGTs. The total system costs are lower for H2-CCGTs so this is the technology we should consider.

**Comment by oruhnau** [comment]
I now also implemented to option to disable h2 pipelines on this branch. 

This pushes back H2 storage quite a bit: it is now only built in Denmark (3.2 GW_el H2 electrolysis, 2.5 GW_th H2-CCGT, 140 GWh H2 storage volume.

Seems sensible to me.

File: scripts/solve_network_together.py [hunk 1]
```diff
@@ -260,6 +260,9 @@ def add_entsoe_demand(n):
     )
     n.stores.loc[store_i, "capital_cost"] = store_cost 
 
+    if not snakemake.config["scenario"]["h2_pipelines"]:
```
Comment (lisazeyen) [inline scripts/solve_network_together.py]: I would suggest combining this with setting the storage cost to medium pressure tank in the lines before or at least making the hydrogen storage costs in the background system another option in the config as well.
So e.g. having the switch in the config `hydrogen_backbone: True/False` and if `True` we have cheap underground storage + pipelines or we add another switch only for the storage like `h2_underground_store: True/False`

**Comment by lisazeyen** [comment]
@oruhnau Great that you found some cost assumptions! I think CCGT  have higher efficiencies but are actually less flexible which might work less good with renewables but let's first stick with the CCGTs which can run with H2.

Ok, interesting that the H2 storage is so sensible to the pipelines since the storage costs are the same in every country without underground storage. Currently, I have no good intuition why this is the case.

File: scripts/solve_network_together.py (hunk 1 above)
Comment (oruhnau) [reply scripts/solve_network_together.py]: Good idea. I can implement this tomorrow

**Comment by oruhnau** [comment]
@oruhnau Great that you found some cost assumptions! I think CCGT have higher efficiencies but are actually less flexible which might work less good with renewables but let's first stick with the CCGTs which can run with H2.

I think H2 will only be used for long-duration storage and be complemented by short-duration battery storage. So I wouldn't worry about felxibility of H2 turbines too much.

Ok, interesting that the H2 storage is so sensible to the pipelines since the storage costs are the same in every country without underground storage. Currently, I have no good intuition why this is the case.

My intuition would be that electrolyzers and H2 turbines were build primarily to tap into cheap transmission via H2 pipelines and only as a side effect, storage became attractive (after already bearing converion losses).

Maybe another source to check for efficiency changes https://www.mdpi.com/1996-1073/15/12/4485

**Comment by oruhnau** [comment]
@lisazeyen I could not find any evidence on efficiency changes in the MDPI article neither. So I would assume the same efficiency for now.

**Pull request merged**